src/main/java/org/zalando/logbook/Markable.java

```diff
@@ -0,0 +1,39 @@
+package org.zalando.logbook;
+
+/*
+ * #%L
+ * Logbook
+ * %%
+ * Copyright (C) 2015 Zalando SE
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+import javax.servlet.ServletRequest;
+
+interface Markable extends Named {
```

Comment:
I don't see the need for having this as an interface with default methods instead of protected methods in `OnceFilter`. Default methods are nice if a subclass might have a more optimized implementation, but here it just makes the code harder to follow.